### DISCUSSION TITLE
json schema store match current api

### DISCUSSION THREAD

**Discussion by FrederickEngelhardt** [swc-project/swc#5429] [Q&A] [answered]
### Describe the feature

Hello all, thanks again for maintaining this codebase!

**Request:**
When using "http://json.schemastore.org/swcrc" I am unable to get completion for "minify" and other sub directory commands inside of "jsc".

Is it possible to auto upload these schema changes (per version) so we can point to the matching schema IE "http://json.schemastore.org/swcrc/1.2.224"

Again I'm not familiar with schemastore.org, but am curious if versioning the json is possible and if these jsc parameters can be added

Example documentation:
https://swc.rs/docs/configuration/minification#jscminifycompress, when implementing this, it will show in the json as unallowed.

<img width="538" alt="Screen Shot 2022-08-09 at 7 08 46 PM" src="https://user-images.githubusercontent.com/31741411/183794835-f2c3bcd3-14cd-48f2-831c-ad25de9ebfdc.png">

```json
{
  "$schema": "http://json.schemastore.org/swcrc",
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": true
    },
    "target": "es2020",
    "loose": false,
    "minify": {
      "compress": {
        "unused": true,
        "dead_code": true
      }
    }
  },
  "module": {
    "type": "es6"
  },
  "minify": true,
  "isModule": false,
  "sourceMaps": true
}
```

### Babel plugin or link to the feature description

auto publish to schemastore.org and include jsc types

**Comment by kwonoj** [comment, marked as answer]
We don't own schemastore for the json and afaik it's community driven requires PR approval that we can't easily automate.
